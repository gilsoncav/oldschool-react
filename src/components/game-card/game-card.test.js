import GameCard from './game-card';

describe('GameCard component', () => {
  it('renders correctly', () => {
    const snapshot = snapRender(<GameCard onClick={() => {}} />);
    expect(snapshot).toMatchSnapshot();
  });
});
