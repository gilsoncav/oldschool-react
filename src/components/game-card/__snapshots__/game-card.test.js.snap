// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameCard component renders correctly 1`] = `
.c0 {
  outline: none;
  background-color: #373e49;
  color: #ff6347;
  border-radius: 15px;
  height: inherit;
  padding: 10px;
  margin: 10px;
  text-align: center;
}

.c0:hover {
  background-color: #2c313a;
  cursor: pointer;
}

.c0 div:nth-child(1) div:nth-child(1) {
  width: inherit !important;
  height: 250px !important;
}

.c0 div:nth-child(2) > span,
.c0 div:nth-child(3) > span {
  overflow: hidden;
  text-overflow: ellipsis;
  display: -webkit-box;
  -webkit-box-orient: vertical;
  -webkit-line-clamp: 1;
}

.c0 > div:nth-child(4) > span {
  overflow: hidden;
  text-overflow: ellipsis;
  display: -webkit-box;
  -webkit-box-orient: vertical;
  -webkit-line-clamp: 3;
}

<div
  className="c0"
  onClick={[Function]}
  onKeyPress={[Function]}
  role="button"
  tabIndex={0}
>
  <div>
    <div>
      <strong>
        No video available for this game.
      </strong>
    </div>
  </div>
  <div>
    <strong>
      Title: 
    </strong>
    <span>
      Super Mario Kart
    </span>
  </div>
  <div>
    <strong>
      Platform: 
    </strong>
    <span>
      Super Nintendo Entertainment System
    </span>
  </div>
  <div>
    <strong>
      Overview: 
    </strong>
    <span>
      The best kart game in the world, y' now
    </span>
  </div>
</div>
`;
